Content.Client/Medical/CrewMonitoring/CrewMonitoringWindow.xaml.cs

```diff
@@ -209,6 +209,10 @@ private void PopulateDepartmentList(IEnumerable<SuitSensorStatus> departmentSens
             {
                 specifier = new SpriteSpecifier.Rsi(new ResPath("Interface/Alerts/human_crew_monitoring.rsi"), "dead");
             }
+            
+            // Floofstation - unique state for IPC
+            else if (sensor.IsDischarged)
```

Comment:
No blank lines between block-less `if` and `else`, please.
```suggestion
            else if (sensor.IsDischarged) // Floofstation - unique state for IPC
```

Reply:
Ah, yeah. That makes sense. I originally was going to use a block statement like the if but it bothered me to do a block for one line of code so I switched away from it.